src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -127,11 +136,15 @@ private List<BeneficialOwnerIndividualDto> getBeneficialOwnersIndividualSubmissi
         }
 
         for (BeneficialOwnerIndividualDto beneficialOwner : submissionDto.getBeneficialOwnersIndividual()) {
-            String noTrustsMessage = "No trusts exist for this filing but a trust id is provided for BO Individual "
-                    + beneficialOwner.getFirstName() + " " + beneficialOwner.getLastName();
 
-            List<TrustDataDto> trustData = getTrustData(submissionDto, beneficialOwner.getTrustIds(), noTrustsMessage);
-            beneficialOwner.setTrustData(convertTrustDataToString(trustData));
+            if (!isTrustsSubmissionThroughWebEnabled) {
+                String noTrustsMessage = "No trusts exist for this filing but a trust id is provided for BO Individual "
+                        + beneficialOwner.getFirstName() + " " + beneficialOwner.getLastName();
+
+                List<TrustDataDto> trustData = getTrustData(submissionDto, beneficialOwner.getTrustIds(),
```

Comment:
Just need to be sure to not lose the validation present in the 'getTrustData' method when the new trust journey is active. This will probably move into the API validation for trusts in a later PR. @ch-tstrickland is aware of this.

Reply:
Added as a comment on our validation ticket:
https://companieshouse.atlassian.net/browse/ROE-1549?focusedCommentId=203557